tsconfig.json

```
@@ -21,5 +21,5 @@
 		"rootDir": "./src",
 		"outDir": "./dist"
 	},
-	"include": ["./src/**/*.ts", "./src/**/*.json"]
+	"include": ["./src/**/*.ts", "./src/**/*.json", "jest.unit.config.js"]
```

Please check if this works, so that we include all the test configs (in case we decide to add in the future).
```suggestion
	"include": ["./src/**/*.ts", "./src/**/*.json", "jest**config.js"]
```

---

Applied.